applications/openshift/api-server/api_server_tls_security_profile_not_old/rule.yml

```diff
@@ -0,0 +1,60 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Check if API server is configured to use Old tlsSecurityProfile"
+
+description: |-
+    The configuration <tt>tlsSecurityProfile</tt> specifies TLS configurations
+    to be used while establishing connections with the externally exposed
+    servers. Though secure transport mode is used for establishing connections,
+    the protocols used may not always be strong enough to avoid interception and
+    manipulation of the data in transport. TLS Security profile Old should be
+    avoided, as it supports vulnerable protocols, ciphers, and algorithms which
+    could lead to security breaches.
+
+    Update tlsSecurityProfile from Old to Intermediate using the following command:
+    {{% raw %}}<pre>oc patch apiservers.config.openshift.io cluster --type 'json' --patch '[{"op": "add", "path": "/spec/tlsSecurityProfile/intermediate", "value": {}}, {"op": "replace", "path": "/spec/tlsSecurityProfile/type", "value": "Intermediate"}, {"op": "remove", "path": "/spec/tlsSecurityProfile/old"}]'</pre>{{% endraw %}}
+
+    For more information, follow
+    {{{ weblink(link="https://docs.openshift.com/container-platform/latest/security/tls-security-profiles.html",
+                text="the relevant documentation") }}}.
+
+rationale: |-
+    The authenticity and integrity of the container platform and communication
+    between nodes and components must be secure. If an insecure protocol,
+    cipher, or algorithms is used, during transmission of data, the data can be
+    intercepted and manipulated. To thwart the manipulation of the data during
+    transmission secure protocol, cipher and algorithms must be used.
+
+severity: medium
+
+identifiers:
+    cce@ocp4: CCE-86223-5
+
+references:
+    nist: CM-6
```

Comment:
Please rather add SC-8 and SC-8(1) (in all rules)

Reply:
comment incorporated.